public/leaderboardinfo.php

```diff
@@ -44,7 +44,7 @@
 $consoleName = $lbData['ConsoleName'];
 $forumTopicID = $lbData['ForumTopicID'];
 
-$pageTitle = "Leaderboard: $lbTitle ($gameTitle)";
+$pageTitle = "$lbTitle ($gameTitle, $consoleName)";
```

Comment:
Recommend mimicking achievement info page:
```
$pageTitle = "$lbTitle in $gameTitle ($consoleName)";
```

Reply:
Updated to your example in latest.